webview-ui/src/components/chat/checkpoints/CheckpointMenu.tsx

```diff
@@ -56,14 +59,16 @@ export const CheckpointMenu = ({ ts, commitHash }: CheckpointMenuProps) => {
 				</PopoverTrigger>
 				<PopoverContent align="end" container={portalContainer}>
 					<div className="flex flex-col gap-2">
-						<div className="flex flex-col gap-1 group hover:text-foreground">
-							<Button variant="secondary" onClick={onPreview}>
-								Restore Files
-							</Button>
-							<div className="text-muted transition-colors group-hover:text-foreground">
-								Restores your project's files back to a snapshot taken at this point.
+						{!isCurrent && (
+							<div className="flex flex-col gap-1 group hover:text-foreground">
+								<Button variant="secondary" onClick={onPreview}>
+									Restore Files
```

Comment:
This action does nothing if this checkpoint is the current checkpoint, so let's hide it.